client/dashboard/task-list/tasks/shipping/index.js

```diff
@@ -174,6 +174,11 @@ class Shipping extends Component {
 				),
 				content: (
 					<ShippingRates
+						buttonText={
+							[ 'US', 'GB', 'CA', 'AU' ].includes( countryCode )
```

Comment:
Nitpick: We have a few of these country code arrays in here now (for example `visible: [ 'US', 'GB', 'CA', 'AU' ].includes( countryCode ),` on the plugins step).

If WooCommerce Services/Shipping or ShipStation adds support for more countries, we'll need to make sure all the arrays are updated correctly. Should we instead have an array of countries for WCS, an array for ShipStation, and then use those in the correct places?

Reply:
Agreed.  I updated this a bit in https://github.com/woocommerce/woocommerce-admin/pull/3400/commits/a7aed344ed0a5030a57e6549307b0f1f59afe345 to determine task visibility by the plugins so we can consolidate the country checks down to a single area.  I think these steps are actually dependent on plugins so I'm thinking this makes more sense.

Let me know if you think this makes sense or if you have something else in mind.

Reply:
Yeah I think that's a great way to handle it 👍. I'll go ahead and re-test this, and then merge it, since I believe you are AFK now.